tagmanager/src/tm_source_file.h

```diff
@@ -14,14 +14,12 @@
 #include <glib.h>
 #include <glib-object.h>
 
+#include "tm_parser.h"
```

Comment:
Yeah, that's an option too - on the other hand lang in tm_source_file.h and filetypes.h is public, including docstring, and then its value is just some magic constant.